source/module.coffee

```diff
@@ -1,3 +1,4 @@
+{ log } = Space.Configuration if Meteor.isServer
```

Comment:
Does this really work? I wonder where we are making the configuration static? Normally you can access the config via `@Configuration` inside a module method.